sdk/trts/trts_switchless.cpp

```diff
@@ -35,23 +35,19 @@
 #include "global_data.h"
 #include "util.h"
 
-extern "C" {
-
 struct sl_uswitchless;
```

Comment:
This line seems unnecessary and could be removed.

Reply:
The struct line doesn't affect the extern "C" / static selection. I'd prefer to keep that change separate.